tests/test_object_model/test_tipping_points.py

```diff
@@ -0,0 +1,92 @@
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import pytest
+import tomli
+from flood_adapt.api.static import read_database
+from flood_adapt.config import set_system_folder
+
+from flood_adapt.object_model.tipping_point import TippingPoint
+from flood_adapt.object_model.interface.tipping_points import ITipPoint
+
+
+@pytest.fixture
+def setup_database():
+    # Mock the database setup or configure a test database
+    read_database(
+        rf"C:\\Users\\morenodu\\OneDrive - Stichting Deltares\\Documents\\GitHub\\FloodAdapt-Database",
+        "charleston_full",
+    )
+    set_system_folder(
+        rf"C:\\Users\\morenodu\\OneDrive - Stichting Deltares\\Documents\\GitHub\\FloodAdapt-Database\\system"
+    )
```

Comment:
This already exists in conftest.py! 

You can access the test database like this (the fixtures you can use are at the bottom of conftest.py):
```
def test_tipping_point_creation(test_db):
  test_db.events.get()
```